tests/pfcwd/conftest.py

```diff
@@ -244,3 +244,28 @@ def pfcwd_pause_service(ptfhost):
         needs_resume["garp_service"] = False
 
     logger.debug("pause_service needs_resume {}".format(needs_resume))
+
+
+@pytest.fixture(scope="function", autouse=False)
+def skip_pfcwd_higher_speeds(
+        duthosts,
+        enum_rand_one_per_hwsku_frontend_hostname,
+        setup_pfc_test):
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    if duthost.facts["asic_type"] != "cisco-8000":
+        yield
```

Comment:
@rraghav-cisco  lets add this:


```
if duthost.facts["asic_type"] != "cisco-8000" and not duthost.get_facts().get("modular_chassis"):
   return
```

Also when modifying this, could you quickly confirm if we need the `yield` here?

Reply:
@auspham , I have updated the code.
Yes, without yield we run into a big pytest-related tracebacks. I am not sure what the error is about. It goes away if yield is put back.